packages/next-server/server/next-server.ts

```diff
@@ -285,6 +285,7 @@ export default class Server {
     query: ParsedUrlQuery = {},
     opts: any,
   ) {
+    pathname = pathname.replace(/\/index/, '') || '/'
```

Comment:
Wait, what happens if the path is like, `/indexer`? Should you anchor this?

Reply:
`/\/index/$`